connaisseur/validate.py

```diff
@@ -66,7 +66,9 @@ def get_trusted_digest(host: str, image: Image, policy_rule: dict):
     return digests.pop()
 
 
-def process_chain_of_trust(host: str, image: Image, req_delegations: list):
+def process_chain_of_trust(
+    host: str, image: Image, req_delegations: list
+):  # pylint: disable=too-many-branches
```

Comment:
Maybe it would be better if we refactored the function into more digestible (haha) blocks, instead of turning of linting ;)